IPython/core/debugger.py

```diff
@@ -620,7 +620,11 @@ def do_where(self, arg):
         Take a number as argument as an (optional) number of context line to
         print"""
         if arg:
-            context = int(arg)
+            try:
+                context = int(arg)
+            except ValueError:
+                self.stdout.write('\n' + self.shell.get_exception_only())
```

Comment:
Should this be `self.error(err)`, looking at lines above?

Reply:
Defined as https://github.com/python/cpython/blob/c606624af8d4cb3b4a052fb263bb983b3f87585b/Lib/pdb.py#L459

Reply:
I changed it